packages/storybook/src/stories/components/typography/punchline/punchline.stories.tsx

```diff
@@ -1,17 +1,54 @@
 import React from 'react';
-
-import { Text } from '@t3n/components';
-
-import { storyContainerDecorator } from '../../../../utils/decorators';
+import {
+  Punchline,
+  Grid,
+  GridItem,
+  VisualSection,
+  Box,
+  Heading
+} from '@t3n/components';
+import { withKnobs } from '@storybook/addon-knobs';
+import { storyContainerContentDecorator } from '../../../../utils/decorators';
 
 export default {
   title: 'Components|Typografie/Punchline',
-  component: Text,
-  decorators: [storyContainerDecorator]
+  component: Text
 };
 
-export const defaultStory = () => <Text>todo</Text>;
+export const defaultStory = () => <Punchline>Punchline</Punchline>;
 
 defaultStory.story = {
-  name: 'Default'
+  name: 'Default',
+  decorators: [withKnobs, storyContainerContentDecorator]
+};
+
+export const variantStory = () => (
+  <>
+    <Heading as="h3" ml={3}>
+      Punchline 50% width
+    </Heading>
+    <Grid>
+      <GridItem width={[1, 1, 1 / 2, 1 / 2]}>
+        <Box ml={4}>
+          <Punchline>Test für Punchline mit Zeilenumbruch</Punchline>
+        </Box>
+      </GridItem>
+    </Grid>
+    <Heading as="h3" ml={3} mt={8}>
+      Punchline in Visual-Section 100% width
+    </Heading>
+    <Grid>
+      <GridItem width={[1]}>
```

Comment:
Gerne jede einzelne Ansicht als eine Story, also die Punchline in der VisualSection zu zeigen ist ja ein komplett anderer Case als zB der Umbruch

Reply:
alles klar